### ISSUE TITLE
Errors during linking

### ISSUE THREAD

**Issue by Zosoworld** [Xerbo/aptdec#12]
**Describe the bug**
Attempting the installation method shown in the README, the project fails in the linking process:

```
[ 10%] Linking C executable aptdec
/usr/bin/ld: CMakeFiles/aptdec.dir/src/main.c.o:(.bss+0xc0): multiple definition of `TempPalette'; CMakeFiles/aptdec.dir/src/color.c.o:(.data+0xc0): first defined here
/usr/bin/ld: CMakeFiles/aptdec.dir/src/main.c.o:(.bss+0x0): multiple definition of `PrecipPalette'; CMakeFiles/aptdec.dir/src/color.c.o:(.data+0x0): first defined here
/usr/bin/ld: CMakeFiles/aptdec.dir/src/pngio.c.o:(.bss+0x20): multiple definition of `PrecipPalette'; CMakeFiles/aptdec.dir/src/color.c.o:(.data+0x0): first defined here
/usr/bin/ld: CMakeFiles/aptdec.dir/src/pngio.c.o:(.bss+0xe0): multiple definition of `TempPalette'; CMakeFiles/aptdec.dir/src/color.c.o:(.data+0xc0): first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/aptdec.dir/build.make:226: aptdec] Error 1
make[1]: *** [CMakeFiles/Makefile2:95: CMakeFiles/aptdec.dir/all] Error 2
make: *** [Makefile:171: all] Error 2
```

**To Reproduce**
Using Fedora 33:
```
sudo dnf install cmake git gcc libsndfile-devel libpng-devel
git clone https://github.com/Xerbo/aptdec.git && cd aptdec
mkdir build && cd build
cmake -DCMAKE_BUILD_TYPE=Release ..
make
```

**Expected behaviour**
The software is linked correctly.

**Link to audio**
None

**Build information**
13d98107ac856863d0de9a39a4df657e0ff29f64
Using CMake

**Comment by Xerbo** [comment]
This should of been fixed in 86b8d61

**Comment by Xerbo** [comment]
Closing due to inactivity
